sashimi/utilities.py

```diff
@@ -33,3 +35,36 @@ def get_last_parameters(parameter_queue, timeout=0.0001):
         except Empty:
             break
     return params
+
+
+# TODO: Consider this for ScopeCuisine
+class SpeedyArrayBuffer:
+    """
+    Buffer for large data arrays based on numpy and using ctypes for speedy copy of data
+    """
+
+    def __init__(self, size=None, *args, **kwargs):
+        """
+        Create a data object of the appropriate size.
+        """
+        super().__init__(**kwargs)
+        self.np_array = np.ascontiguousarray(
+            np.empty(size // 2)
```

Comment:
why did you remove the dtype here? I think it's necessary, this initializes float64 arrays whereas cameras use uint16